tools/check_and_install_pyodide_build.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import subprocess as sp
+import sys
+
+# Note for maintainers:
+#   Update the following variables when you want to update the version of the
+#   pyodide-build version used in the build process.
+#   If you want to make breaking changes in pyodide-build, it is also useful to
+#   change the repository URL to your fork to test the changes are working as
+#   expected.
+PYODIDE_BUILD_REPO: str = "https://github.com/pyodide/pyodide-build"
+PYODIDE_BUILD_COMMIT: str = "fac0109aa2acf14469320b049d710dd42639bf94"  # v0.27.3
```

Comment:
I'd prefer if we could put this in `Makefile.envs` or failing that into `pyproject.toml`. I don't want to add more locations where configuration lives if we can help it.

Reply:
Makes sense. I moved it to Makefile.envs.